tests/test_hsm_signer.py

```diff
@@ -114,6 +114,11 @@ def setUpClass(cls):
         slot = lib.getSlotList(tokenPresent=True)[0]
         lib.initToken(slot, so_pin, token_label)
 
+        # suddenly there are two slots (why?), use label to filter in tests
```

Comment:
> suddenly there are two slots (why?)

I don't quite understand this comment here. I do have an idea, but only because I read the commit message too. Please rephrase or remove.